rpcs3/Emu/RSX/GL/rsx_gl_texture.cpp

```diff
@@ -463,9 +463,13 @@ namespace rsx
 
 			glTexParameteri(m_target, GL_TEXTURE_COMPARE_FUNC, gl_tex_zfunc[tex.zfunc()]);
 
-			// TODO: Handle texture bias
-			glTexParameteri(m_target, GL_TEXTURE_MIN_LOD, (tex.min_lod() >> 8));
-			glTexParameteri(m_target, GL_TEXTURE_MAX_LOD, (tex.max_lod() >> 8));
+			glTexEnvi(GL_TEXTURE_FILTER_CONTROL, GL_TEXTURE_LOD_BIAS, (GLint)tex.bias());
+			
+			if (m_target != GL_TEXTURE_RECTANGLE)
```

Comment:
I think native TEXTURE_RECTANGLE support was dropped in favor of emulated unnormalized access in the shaders. m_target can only be 1D, 2D, 3D or CUBE now if i'm not mistaken. I remember having a conversation with vlj about this.